src/analysis/normed_space/inner_product.lean

```diff
@@ -1024,6 +1024,21 @@ by simp_rw [sum_inner, inner_sum, real_inner_smul_left, real_inner_smul_right,
             h₁, h₂, zero_mul, mul_zero, finset.sum_const_zero, zero_add, zero_sub, finset.mul_sum,
             neg_div, finset.sum_div, mul_div_assoc, mul_assoc]
 
+/-- The inner product with a fixed left element, as a continuous linear map.  This can be upgraded
+to a continuous map which is jointly conjugate-linear in the left argument and linear in the right
+argument, once (TODO) conjugate-linear maps have been defined. -/
+def inner_left (v : E) : E →L[𝕜] 𝕜 :=
```

Comment:
My instinct would have been to call this `inner_right`, after the element that is not fixed, but maybe that's just me. Maybe there's a similar definition elsewhere that sets a precedent...?

Reply:
Thanks, I made the change.  I have no strong opinion here, will defer to consensus.